04-reporting.Rmd

```diff
@@ -431,6 +430,40 @@ some latex $\\alpha$"
 
 `kableExtra` is another powerful alternative
 
+### Table captions
+
+Table captions can by passing a character string to `caption = ` argument within apa_table function (e.g. `apa_table(descriptives, caption = "My table caption")`). However, as with figures we strongly recommend that you use [text references](https://bookdown.org/yihui/bookdown/markdown-extensions-by-bookdown.html#text-references)---a feature of the `bookdown` package---instead.
+A text reference consists of a unique label and the corresponding text that it represents.
+Text references can be defined as follows:
+
+```
+(ref:my-table-caption) This is a caption for my table.
+```
+
+The text reference can now be inserted in the chunk options (or anywhere else), e.g. `apa_table(descriptives, caption = "my-table-caption")`.
```

Comment:
Unless I'm mistaken, I think the text references needs to be called as `caption = "(ref:my-table-caption)"`.

Reply:
Yes you're right